Firebase/Database/third_party/SocketRocket/FSRWebSocket.m

```diff
@@ -1356,6 +1356,10 @@ - (void)_sendFrameWithOpcode:(FSROpCode)opcode data:(id)data;
 
     NSAssert(data == nil || [data isKindOfClass:[NSData class]] || [data isKindOfClass:[NSString class]], @"Function expects nil, NSString or NSData");
 
+    if (data == nil) {
```

Comment:
This is fine as is, but if you want to spend more time on this then I would move this one line up and change the assert to no check for nil values.